pkg/operator/ceph/controller/spec.go

```diff
@@ -75,22 +75,24 @@ var (
 	cronLogRotate = `
 CEPH_CLIENT_ID=%s
 PERIODICITY=%s
-LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
-
-if [ -z "$PERIODICITY" ]; then
-	PERIODICITY=24h
-fi
+LOG_ROTATE_CEPH_FOLDER=/etc/logrotate.d/ceph
+DAEMON_LOG_FILE=/var/log/ceph/$CEPH_CLIENT_ID.log
+LOG_MAX_SIZE=%s
 
 # edit the logrotate file to only rotate a specific daemon log
 # otherwise we will logrotate log files without reloading certain daemons
 # this might happen when multiple daemons run on the same machine
-sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"
+sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FOLDER"
+
+# replace default daily with given user input
+sed --in-place "s/daily/$PERIODICITY/g" "$LOG_ROTATE_CEPH_FOLDER"
+
+# append maxsize at 4th line ater the $PERIODICITY with 4spaces to maintain indentation
```

Comment:
`at 4th line`, I didn't understand,

Can you please elaborate in the comment

Reply:
we are adding `maxsize $LOG_MAX_SIZE` at the 4th line of the logrotate config file with 4 spaces to maintain indentation

Reply:
adding maxsize $LOG_MAX_SIZE at the 4th line of the logrotate config file with 4 spaces to maintain indentation

This line makes more understandability.